airbyte-integrations/connectors/source-firebase-realtime-database/integration_tests/abnormal_state.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
You can remove this file since we don't have an incremental stream

Reply:
fixed in https://github.com/airbytehq/airbyte/pull/18029/commits/ea934df62a530ad26eceaeeb4c7a344e49938806